From 5221fc21ee5c3220af5bcc58d4a6e6892408af99 Mon Sep 17 00:00:00 2001
From: Li Wencheng <liwencheng@phytium.com.cn>
Date: Fri, 26 Apr 2024 14:52:58 +0800
Subject: [PATCH 15/67] PHYTIUM: net/phytmac: Adjust the code style

Fix spelling mistakes and replace the default value
with a macro definition.

Mainline: NA
Signed-off-by: Li Wencheng <liwencheng@phytium.com.cn>
Signed-off-by: Wang Yinfeng <wangyinfeng@phytium.com.cn>
Change-Id: Icc51d45f0fb10eb8dad8aad59f839f2c0406653c

Link: https://gitee.com/phytium_embedded/phytium-linux-kernel/commit/6848fd7516a4ab90e013a42513bb8c8b6a8b6cf8
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/net/ethernet/phytium/phytmac_main.c | 4 +++-
 drivers/net/ethernet/phytium/phytmac_v1.c   | 4 ++--
 drivers/net/ethernet/phytium/phytmac_v1.h   | 8 ++++++++
 3 files changed, 13 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/phytium/phytmac_main.c b/drivers/net/ethernet/phytium/phytmac_main.c
index 07f8476946bc..ca1a03fa3552 100644
--- a/drivers/net/ethernet/phytium/phytmac_main.c
+++ b/drivers/net/ethernet/phytium/phytmac_main.c
@@ -1416,10 +1416,12 @@ void phytmac_pcs_link_up(struct phylink_pcs *pcs, unsigned int mode,
 			    phy_modes(interface), speed, duplex);
 	hw_if->pcs_linkup(pdata, interface, speed, duplex);
 }
+
 static const struct phylink_pcs_ops phytmac_pcs_phylink_ops = {
 	.pcs_config = phytmac_pcs_config,
 	.pcs_link_up = phytmac_pcs_link_up,
 };
+
 static struct phylink_pcs *phytmac_mac_select_pcs(struct phylink_config *config,
 						   phy_interface_t interface)
 {
@@ -1766,7 +1768,7 @@ static int phytmac_open(struct net_device *ndev)

 	ret = phytmac_phylink_connect(pdata);
 	if (ret) {
-		netdev_err(ndev, "phylink connet failed,(error %d)\n",
+		netdev_err(ndev, "phylink connect failed,(error %d)\n",
 			   ret);
 		goto reset_hw;
 	}
diff --git a/drivers/net/ethernet/phytium/phytmac_v1.c b/drivers/net/ethernet/phytium/phytmac_v1.c
index ec95c6c79b06..4a690b6a96cf 100644
--- a/drivers/net/ethernet/phytium/phytmac_v1.c
+++ b/drivers/net/ethernet/phytium/phytmac_v1.c
@@ -414,7 +414,7 @@ static int phytmac_init_hw(struct phytmac *pdata)
 	PHYTMAC_WRITE(pdata, PHYTMAC_DCONFIG, dmaconfig);

 	if (pdata->capacities & PHYTMAC_CAPS_TAILPTR)
-		PHYTMAC_WRITE(pdata, PHYTMAC_TAIL_ENABLE, 0x80000001);
+		PHYTMAC_WRITE(pdata, PHYTMAC_TAIL_ENABLE, PHYTMAC_BIT(TXTAIL_ENABLE));

 	if (phy_interface_mode_is_8023z(pdata->phy_interface))
 		phytmac_pcs_software_reset(pdata, 1);
@@ -945,7 +945,7 @@ static void phytmac_tx_start(struct phytmac_queue *queue)

 	if (pdata->capacities & PHYTMAC_CAPS_TAILPTR)
 		PHYTMAC_WRITE(pdata, PHYTMAC_TAILPTR(queue->index),
-			      BIT(31) | queue->tx_tail);
+			      PHYTMAC_BIT(TXTAIL_UPDATE) | queue->tx_tail);

 	if (pdata->capacities & PHYTMAC_CAPS_START)
 		PHYTMAC_WRITE(pdata, PHYTMAC_NCTRL,
diff --git a/drivers/net/ethernet/phytium/phytmac_v1.h b/drivers/net/ethernet/phytium/phytmac_v1.h
index d8de2c26cab4..2cabadfed2f1 100644
--- a/drivers/net/ethernet/phytium/phytmac_v1.h
+++ b/drivers/net/ethernet/phytium/phytmac_v1.h
@@ -356,6 +356,14 @@ extern struct phytmac_hw_if phytmac_1p0_hw;
 #define PHYTMAC_VLAN_ID_INDEX			10
 #define PHYTMAC_VLAN_ID_WIDTH			1

+/* Bitfields in TAILPTR */
+#define PHYTMAC_TXTAIL_UPDATE_INDEX		31 /* Update tx tail */
+#define PHYTMAC_TXTAIL_UPDATE_WIDTH		1
+
+/* Bitfields in TAIL_ENABLE */
+#define PHYTMAC_TXTAIL_ENABLE_INDEX		0 /* Enable tx tail */
+#define PHYTMAC_TXTAIL_ENABLE_WIDTH		1
+
 #define PHYTMAC_TSEC_WIDTH (PHYTMAC_SECH_WIDTH + PHYTMAC_SECL_WIDTH)
 #define SEC_MAX_VAL (((u64)1 << PHYTMAC_TSEC_WIDTH) - 1)
 #define NSEC_MAX_VAL ((1 << PHYTMAC_NSEC_WIDTH) - 1)
--
2.50.1
